src/gui/application.cpp

```diff
@@ -282,6 +273,19 @@ Application::Application(int &argc, char **argv)
     setupLogging();
     setupTranslations();
 
+#if defined(OC_PLUGIN_DIR) && defined(Q_OS_LINUX)
+    qCInfo(lcApplication) << "Added extra plugin search path:" << extraPluginDir;
```

Comment:
We could get rid of the extra ifdef if we just log the `libraryPath()`

Reply:
Right. You mentioned that before.